packages/core-commands/src/admin-navigation-commands.js

```diff
@@ -45,7 +44,7 @@ export function useAdminNavigationCommands() {
 		label: __( 'Patterns' ),
 		icon: symbol,
 		callback: ( { close } ) => {
-			if ( isTemplatesAccessible && isBlockBasedTheme ) {
+			if ( isTemplatesAccessible ) {
```

Comment:
Makes sense. I just reused this comment:

https://github.com/WordPress/gutenberg/blob/08a81568742595e6943b83fdfff55217e9ef3f97/packages/patterns/src/components/patterns-manage-button.js#L38-L40

Reply:
I'm not 100% sure that comment captures the situation here.

The `isTemplatesAccessible` value is set via `useIsTemplatesAccessible` which checks `read` permissions for `templates`. Additionally, it should probably refer to this as a command rather than a link.

Reply:
I updated it with a0e4add, but I'm not sure if I'm communicating it well 😅